### PR TITLE
Added Question and Answer to the Q&A - Please confirm with the P

### PR DESCRIPTION
**Pull request by Kenneth-Lindgren**
It is a common request to avoid having the guest/b2b accounts added in the host tenant, to help our customers I have added this Q/A questions, to make it clear. 
Please ask PG to review to ensure it does not conflict with any new functionality.  
If this is possible by new functionality, please add this as a clear point in to documentation to avoid confusion. 
Thanks.

### CODE REVIEW

**Comment by Court72** [comment]
@msmimart

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/external-id/direct-federation.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ Currently, the Microsoft Entra SAML/WS-Fed federation feature doesn't support se
 
 You need to be an [External Identity Provider Administrator](~/identity/role-based-access-control/permissions-reference.md#external-identity-provider-administrator) or a [Global Administrator](~/identity/role-based-access-control/permissions-reference.md#global-administrator) in your Microsoft Entra tenant to configure a SAML/Ws-Fed identity provider.
 
+**Can we avoid the creation of guest/B2B accounts on our tenant?**
```
Comment (msmimart) [inline docs/external-id/direct-federation.md:111]:
```suggestion
**Does federation eliminate the need to create a guest account in my directory for the B2B collaboration user?**
```

File: docs/external-id/direct-federation.md [hunk 2]
```diff
@@ -108,6 +108,10 @@ Currently, the Microsoft Entra SAML/WS-Fed federation feature doesn't support se
...
+
+Guests accounts will always be need to be created on the teant in order to grant access to applications, roles or membership to security groups or teams in the teanat, regardless of the federation method used.
```
Comment (msmimart) [inline docs/external-id/direct-federation.md:113]: No. A guest account is created for a B2B collaboration user in your directory regardless of the authentication or federation method used. This user object allows you to grant access to applications, assign roles, and define membership in security groups and teams.

**Comment by msmimart** [comment]
Hi @Kenneth-Lindgren, thank you so much for your suggestion, and apologies for the long delay. Please see the suggested edits; if you'd like to make these changes we can go ahead and sign off. Thanks again!

**Review by msmimart — CHANGES_REQUESTED** [review]

File: docs/external-id/direct-federation.md (hunk 2 above)
Comment (msmimart) [inline docs/external-id/direct-federation.md:113]:
```suggestion
No. A guest account is created for a B2B collaboration user in your directory regardless of the authentication or federation method used. This user object allows you to grant access to applications, assign roles, and define membership in security groups and teams.  
```

**Review by msmimart — CHANGES_REQUESTED** [review]

**Comment by Court72** [comment]
@msmimart - I committed your requested changes. Add a `#sign-off` comment if these changes are ready to merge. Thanks!

**Comment by msmimart** [comment]
#sign-off

**Pull request merged**